ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CreateVulnReportPage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
```

Comment:
I moved the `CreateVulnReportPage` under a new directory because we'll have multiple flows for modifying a report (edit, clone, create). So I thought it made sense to group them all under the same directory. Because I moved the file, git isn't showing the proper diff. I'll comment the changes I've made